packages/form_bloc/lib/src/blocs/form/form_bloc_utils.dart

```diff
@@ -374,4 +374,49 @@ class FormBlocUtils {
       fieldBloc.removeFormBloc(formBloc);
     }
   }
+
+  /// See [FormBloc.isValuesChanged]
+  static bool isValuesChanged(
+    Iterable<FieldBloc> fieldBlocs, {
+    bool fallback = false,
+  }) {
+    return fieldBlocs.any((fb) {
+      if (fb is SingleFieldBloc) {
+        return fb.isValueChanged;
```

Comment:
can we move isValueChanged to mixin FieldBloc? so we dont need to check the type

Reply:
The names are different than the methods and I wouldn't want to force the developer to have to implement them if his FieldBloc implementation has no "values". What do you think?